### PR TITLE
Syntax error fixes

### PR DESCRIPTION
**Pull request by chynu**
Before working on adding view elements to start making the refactor code actually run the app, I ran into some syntax errors. App still doesn't work, but the errors I now get are what I expect.

---
This change is [Reviewable](https://reviewable.io/reviews/google/blockly-devtools/97)

### CODE REVIEW

File: src/view/app_view.js [hunk 1]
```diff
@@ -290,8 +290,12 @@ class AppView {
       return tree;
     }
 
-    for (let index in tree) {
+    for (let index = 0; index < tree.length; index++) {
```
Comment (AnmAtAnm) [inline src/view/app_view.js]: Try Array.foreach()

**Review by svouse — APPROVED** [review]

**Comment by svouse** [comment]
:lgtm:

---







*Comments from [Reviewable](https://reviewable.io:443/reviews/google/blockly-devtools/97#-:-Koi9ytsWjhoh2hgk-qn:bnfp4nl)*

File: src/view/navigation_tree.js [hunk 2]
```diff
@@ -194,18 +194,24 @@ class NavigationTree {
     const items = {
       renameElement : {
         label: "Rename",
-        action: () => console.warn("Action undefined");
```
Comment (AnmAtAnm) [inline src/view/navigation_tree.js:197]: Surprised this doesn't work, from the syntax spec on MDN.

**Review by AnmAtAnm — APPROVED** [review]
One minor suggestion.

**Comment by rachel-fenichel** [comment]
Review status: 0 of 4 files reviewed at latest revision, 2 unresolved discussions.

---

*[src/view/app_view.js, line 295 at r1](https://reviewable.io:443/reviews/google/blockly-devtools/97#-KoiA4Gcbag2Bv5vupR4:-KoiA4GdCS_Lgh4YO6Kh:b86v753) ([raw file](https://github.com/google/blockly-devtools/blob/f4068fb941ec6a333b2b270e8b84ae933950367b/src/view/app_view.js#L295)):*
> ```JavaScript
>     for (let index = 0; index < tree.length; index++) {
>       let pair = tree[index];
>       if (pair == undefined) {
> ```

Please figure out what's breaking now, rather than patching around it.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/google/blockly-devtools/97)*

**Comment by chynu** [comment]
Review status: 0 of 4 files reviewed at latest revision, 2 unresolved discussions.

---

*[src/view/app_view.js, line 293 at r1](https://reviewable.io:443/reviews/google/blockly-devtools/97#-Koi9TbEjviuxXnBaz6x-r1-293:-Koio7sC7I_8Tol8VHYu:b-vmhql5) ([raw file](https://github.com/google/blockly-devtools/blob/f4068fb941ec6a333b2b270e8b84ae933950367b/src/view/app_view.js#L293)):*
<details><summary><i>Previously, AnmAtAnm (Andrew n marshall) wrote…</i></summary><blockquote>

Try Array.foreach()
</blockquote></details>

Done. This fixed the `undefined` problem as well.

---

*[src/view/app_view.js, line 295 at r1](https://reviewable.io:443/reviews/google/blockly-devtools/97#-KoiA4Gcbag2Bv5vupR4:-KoioCnm0EUkFyQuMPL1:b-ioq02p) ([raw file](https://github.com/google/blockly-devtools/blob/f4068fb941ec6a333b2b270e8b84ae933950367b/src/view/app_view.js#L295)):*
<details><summary><i>Previously, rachel-fenichel (Rachel Fenichel) wrote…</i></summary><blockquote>

Please figure out what's breaking now, rather than patching around it.
</blockquote></details>

Fixed by switching to `Array.foreach()`.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/google/blockly-devtools/97)*

**Comment by rachel-fenichel** [comment]
:lgtm:

---

Review status: 0 of 4 files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/google/blockly-devtools/97#-:-KoiqQnRUwZgXYgmntTq:bnfp4nl)*

**Comment by AnmAtAnm** [comment]
:lgtm:

---

Review status: 0 of 4 files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/google/blockly-devtools/97#-:-KoiwVoq60hEc7MiCCXD:bnfp4nl)*

**Review by AnmAtAnm — APPROVED** [review]

**Pull request merged**